client/cli/src/commands/runcmd.rs

```diff
@@ -562,13 +563,34 @@ fn interface_str(
 	}
 }
 
-/// Default to verbosity level 0, if none is provided.
+/// Utility error for returning custom errors that implement the std::error::Error trait.
+#[derive(Debug)]
+struct TelemetryParsingError {
+    details: String
+}
+
+impl TelemetryParsingError {
+    fn new(msg: &str) -> TelemetryParsingError {
+        TelemetryParsingError{details: msg.to_string()}
+    }
+}
+
+impl fmt::Display for TelemetryParsingError {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}", self.details)
+    }
+}
+
+impl Error for TelemetryParsingError {
+    fn description(&self) -> &str {
+        &self.details
+    }
+}
+
```

Comment:
Is there a simpler way of having a simple String implement the std::error::Error trait?

Reply:
Why not just `enum TelemetryError { MissingVerbosity }`? And `impl std::error::Error for TelemetryError {}` and `impl Display`?